SPECS/ucx/ucx.spec

```diff
@@ -3,47 +3,56 @@
 %bcond_with    cuda
 %bcond_with    gdrcopy
 %bcond_without ib
+#knem disable until MOFED Driver is avialable
+%bcond_with    knem
 %bcond_without rdmacm
 %bcond_with    rocm
 %bcond_with    ugni
+#xpmem disable until MOFED Driver is avialable
 %bcond_with    xpmem
 %bcond_with    vfs
-
-Summary:        UCX is a communication library implementing high-performance messaging
-Name:           ucx
-Version:        1.15.0
-Release:        1%{?dist}
-License:        BSD
+%bcond_with    mad
+%bcond_with    ze
+%bcond_without mlx5
+
+Name: ucx
+Version: 1.18.0
+Release: 3%{?dist}
+Summary: UCX is a communication library implementing high-performance messaging
+Group: System Environment/Libraries
```

Comment:
Nit-pick: can we keep the spacing we used so far?

Reply:
Fixed indentation